build/libs_mac.txt

```diff
@@ -10,6 +10,7 @@ lib_lightgbm.dylib
 libtensorflow.dylib
 libonnxruntime.dylib
 libtensorflow_framework.1.dylib
+Featurizers.dll
```

Comment:
not .so file?

Reply:
I thought that was weird too. This is what was in the original file in the `aml` branch and there are only `*.dylib` or `*.dll` references in that file.